README.md

```diff
@@ -164,7 +164,7 @@ Java generation is integrated with the existing protobuf toolchain for java proj
                     <configuration>
                         <pluginParameter>lang=java</pluginParameter>
                         <pluginId>java-pgv</pluginId>
-                        <pluginArtifact>com.lyft.protoc-gen-validate:pgv:${pgv.version}:exe:${os.detected.classifier}</pluginArtifact>
+                        <pluginArtifact>com.lyft.protoc-gen-validate:protoc-gen-validate:${pgv.version}:exe:${os.detected.classifier}</pluginArtifact>
```

Comment:
does this need to change to `io.envoyproxy` as well?

Reply:
Fixed.